mne_icalabel/tests/test_features.py

```diff
@@ -0,0 +1,480 @@
+    # compare
+    assert np.allclose(topo, topo_eeglab, equal_nan=True)
+
+
+# ----------------------------------------------------------------------------
+def test_eeg_rpsd_constants():
+    """Test _eeg_rpsd_constants function."""
+    # Raw --------------------------------------------------------------------
+    raw = read_raw(raw_eeglab_path, preload=True)
+    ica = read_ica_eeglab(raw_eeglab_path)
+    ncomp, nfreqs, n_points, nyquist, index, window, subset = _eeg_rpsd_constants(
+        raw, ica
+    )
+
+    constants_eeglab = loadmat(psd_constants_raw_path)["constants"][0, 0]
+    ncomp_eeglab = constants_eeglab["ncomp"][0, 0]
+    nfreqs_eeglab = constants_eeglab["nfreqs"][0, 0]
+    n_points_eeglab = constants_eeglab["n_points"][0, 0]
+    nyquist_eeglab = constants_eeglab["nyquist"][0, 0]
+    index_eeglab = constants_eeglab["index"]
+    window_eeglab = constants_eeglab["window"]
+    subset_eeglab = constants_eeglab["subset"]
+
+    assert ncomp == ncomp_eeglab
+    assert nfreqs == nfreqs_eeglab
+    assert n_points == n_points_eeglab
+    assert nyquist == nyquist_eeglab
+    assert np.allclose(index, index_eeglab - 1)
+
+    # window and subset are not squeezed in matlab
+    assert window_eeglab.shape[0] == 1
+    assert np.allclose(window, window_eeglab[0, :])
+
+    # for subsets, compare if the same elements are in both
+    assert subset_eeglab.shape[0] == 1
+    assert len(set(list(subset)).difference(set(list(subset_eeglab[0, :] - 1)))) == 0
+    assert len(set(list(subset_eeglab[0, :] - 1)).difference(set(list(subset)))) == 0
+
+    # Epochs -----------------------------------------------------------------
+    epochs = read_epochs_eeglab(epo_eeglab_path)
+    ica = read_ica_eeglab(epo_eeglab_path)
+    ncomp, nfreqs, n_points, nyquist, index, window, subset = _eeg_rpsd_constants(
+        epochs, ica
+    )
+
+    constants_eeglab = loadmat(psd_constants_epo_path)["constants"][0, 0]
+    ncomp_eeglab = constants_eeglab["ncomp"][0, 0]
+    nfreqs_eeglab = constants_eeglab["nfreqs"][0, 0]
+    n_points_eeglab = constants_eeglab["n_points"][0, 0]
+    nyquist_eeglab = constants_eeglab["nyquist"][0, 0]
+    index_eeglab = constants_eeglab["index"]
+    window_eeglab = constants_eeglab["window"]
+    subset_eeglab = constants_eeglab["subset"]
+
+    assert ncomp == ncomp_eeglab
+    assert nfreqs == nfreqs_eeglab
+    assert n_points == n_points_eeglab
+    assert nyquist == nyquist_eeglab
+    assert np.allclose(index, index_eeglab - 1)
+
+    # window and subset are not squeezed in matlab
+    assert window_eeglab.shape[0] == 1
+    assert np.allclose(window, window_eeglab[0, :])
+
+    # for subsets, compare if the same elements are in both
+    assert subset_eeglab.shape[0] == 1
+    assert len(set(list(subset)).difference(set(list(subset_eeglab[0, :] - 1)))) == 0
+    assert len(set(list(subset_eeglab[0, :] - 1)).difference(set(list(subset)))) == 0
+
+
+def test_eeg_rpsd_compute_psdmed():
```

Comment:
It seems we don't need this extra test given the below `test_eeg_rpsd`(?)

Reply:
Fine by me, the only added information was that if only `test_eeg_rpsd` fails, then the problem lies within `_eeg_rpsd_format`. But that is not worth an additional test and can be debugged on the fly if `test_eeg_rpsd` ever fails.